src/app/(BottomTabNavigation)/AllCases/EligibilityForm/ConfirmEligibility/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Link, router, useLocalSearchParams } from 'expo-router';
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+import Ex from '../../../../../../assets/cancel-x-icon.svg';
+import Check from '../../../../../../assets/check-circle.svg';
+import Line from '../../../../../../assets/line-vector.svg';
+import Alarm from '../../../../../../assets/noIdea.svg';
+import LittlePerson from '../../../../../../assets/noIdea2.svg';
+import {
+  updateCaseActivity,
+  updateCaseStatus,
+} from '../../../../../supabase/queries/cases';
+import { CaseUid, Eligibility } from '../../../../../types/types';
+
+export default function ConfirmEligibility() {
+  const { caseUid } = useLocalSearchParams<{ caseUid: CaseUid }>();
+
+  const updateActivity = async (active: boolean) => {
+    console.log('case', caseUid);
+    console.log('active', active);
+    if (caseUid !== undefined) {
+      if (!active) {
+        await updateCaseStatus(caseUid, Eligibility.INELIGIBLE);
+        await updateCaseActivity(caseUid, active);
+        router.push({
+          pathname: '/AllCases',
+        });
+      } else {
+        router.back();
+      }
+    }
+  };
+  return (
+    <View style={styles.container}>
+      <View style={styles.centerContainer}>
+        <Text style={styles.titleText}>
+          Do you want to mark your status as ineligible?
+        </Text>
+        <View style={styles.infoContainer}>
+          <Alarm style={{ marginRight: 17, marginTop: 5 }} />
+
+          <Text style={styles.infoText}>
+            Once you indicate ineligibility, this case will become inactive.{' '}
+          </Text>
+        </View>
+        <View style={styles.infoContainer}>
+          <LittlePerson style={{ marginRight: 17, marginTop: 12 }} />
+
+          <Text style={styles.infoText}>
+            You can still view and share case details, but won’t be able to
+            change your eligibility status or file a claim.{' '}
+          </Text>
+        </View>
```

Comment:
For these Icon-Text containers, it'd probably be easiest to style by placing a gap between them, and otherwise not setting their margins with the container. Based off the figma, it seems that the top of these icons should align with the top of the accompanying text.